azerothcore/azerothcore-wotlk
# [Core/Spells]: Fix glyph of arcane shot not refunding mana.

## Changes Proposed:
-  Add arcane shot spell script.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6622
- closes https://github.com/chromiecraft/chromiecraft/issues/1020

## SOURCE:


## Tests Performed:

- Logged in on a hunter applied glyph of arcane shot then shot a dummy with my arcane shot and got 20% of the mana refunded.


## How to Test the Changes:


1. Login on a hunter
2. Apply glyph of arcane shot
3. Use arcane shot on a enemy target
4. Check if you've been refunded 20% of your mana

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
